regression/strings-smoke-tests/java_append_char/test.desc

```diff
@@ -7,3 +7,8 @@ assertion.* file test_append_char.java line 23 .* SUCCESS
 assertion.* file test_append_char.java line 25 .* FAILURE
 --
 non equal types
+--
+Runs indefinitely under symex-driven lazy loading. Unsure why yet; this could
+be a bug, or could simply be variations in order of VCCs / functions perturbing
+the string solver and leading to a long unfruitful search that would actually
+terminate eventually.
```

Comment:
As above: please debug. No throwing-over-the-fence, please.

Reply:
Change also dropped.